src/apis/vaults.ts

```diff
@@ -18,15 +20,30 @@ export interface VaultsAPI {
 }
 
 export class DefaultVaultsAPI {
+    private issueAPI!: DefaultIssueAPI;
     granularity = 5;
 
-    constructor(private api: ApiPromise) { }
+    constructor(private api: ApiPromise) { 
+    }
 
     async list(): Promise<Vault[]> {
         const vaultsMap = await this.api.query.vaultRegistry.vaults.entriesPaged({ pageSize: 1 });
         return vaultsMap.map((v) => v[1]);
     }
 
+    async listIssueRequests(vaultId: AccountId): Promise<IssueRequest[]> {
```

Comment:
Similar to the issue requests for the user, please return a `Map` with the issue ID as the key.